redis/provider.go

```diff
@@ -159,3 +225,9 @@ func GenerateReplicationGroupName(instanceID string) string {
 	encoder := base32.StdEncoding.WithPadding(base32.NoPadding)
 	return strings.ToLower("cf-" + encoder.EncodeToString(out))
 }
+
+// GenerateAuthToken generates a password based on the given seed and the service instance id
+func GenerateAuthToken(seed string, instanceID string) string {
+	sha := sha256.Sum256([]byte(seed + instanceID))
```

Comment:
Presumably we're happy that there's enough entropy in the instance ID to avoid some attack involve re-using credentials for Redis instances owned by new users?

Reply:
Yes, we already use the same method for the RDS broker (although that one uses MD5).